rest/utilities_testing_resttester.go

```diff
@@ -491,3 +496,45 @@ func (rt *RestTester) PutDocWithAttachment(docID string, body string, attachment
 	}
 	return rt.PutDocDirectly(docID, rawBody)
 }
+
+type RawDocResponse struct {
+	Xattrs RawDocXattrs `json:"_xattrs"`
+}
+
+// RawDocXattrs is a non-exhaustive set of xattrs returned by the _raw endpoint. Used for test assertions.
+type RawDocXattrs struct {
+	RawDocXattrsWellKnown
+	RawDocXattrsOthers
+}
+
+// RawDocXattrsWellKnown contains well known fields - see RawDocXattrsOthers for those not known until runtime.
+type RawDocXattrsWellKnown struct {
```

Comment:
I wonder if it makes sense to use the same structure for writing, I think `map[string]any` could include json.RawMessage. Not sure, if you don't want to rewrite this.

Reply:
I don't want a strongly typed struct for writing since I want it to tolerate future xattr changes well, but for ease of access, the struct is used in tests to navigate the known objects.